### PR TITLE
Scheduler v1

### PR DESCRIPTION
**Pull request by crufter**
This will likely remove the need for these code pieces:
- https://github.com/micro/actions/blob/master/src/main.ts
- https://github.com/micro/services/blob/master/event.sh
- https://github.com/micro/services/blob/master/platform/web/handler/github/github.go#L44

### CODE REVIEW

File: platform/scheduler/manager/manager.go [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+	return ret
+}
+
+func (m *manager) updateService(folderPath, commit string, status serviceStatus) error {
+	service := &runtime.Service{
+		Name:    folderPath,
+		Version: commit,
+	}
+	typ := typeFromFolder(folderPath)
+	image := "micro/services"
```
Comment (asim) [inline platform/scheduler/manager/manager.go]: The image is a github docker registry image

File: platform/scheduler/manager/manager.go [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+	switch status {
+	case serviceStatusCreated:
+		opts := []runtime.CreateOption{
+			// create a specific service type
```
Comment (asim) [inline platform/scheduler/manager/manager.go]: Are there any other args we were setting when calling create?

File: platform/scheduler/manager/manager.go [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+		ret = append(ret, filepath.Join(parts[0], parts[1]))
+	}
+	return ret
+}
+
+func (m *manager) updateService(folderPath, commit string, status serviceStatus) error {
+	service := &runtime.Service{
+		Name:    folderPath,
+		Version: commit,
+	}
```
Comment (asim) [inline platform/scheduler/manager/manager.go]: Missing Source and Metadata info

File: platform/scheduler/manager/manager.go (hunk 2 above)
Comment (crufter) [reply platform/scheduler/manager/manager.go]: Dont think so

File: platform/scheduler/manager/manager.go (hunk 3 above)
Comment (crufter) [reply platform/scheduler/manager/manager.go]: Not sure about the source tbh

Comment (asim) [reply platform/scheduler/manager/manager.go]: Source is the full url. When you do `micro ps --platform` it displays the actual source url. This is so you know where the code is.

**Pull request merged**